apparmor.d/groups/runit/runsv

```diff
@@ -0,0 +1,177 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2019-2021 Mikhail Morfikov
+# Copyright (C) 2022 Alexandre Pujol <alexandre@pujol.io>
+# Copyright (C) 2024 Besanon <m231009ts@mailfence.com>
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/3.0>,
+
+include <tunables/global>
+
+@{exec_pathrunsv} = @{bin}/runsv
+profile runsv @{exec_pathrunsv} flags=(attach_disconnected) {
+  include <abstractions/base>
+  include <abstractions/bus-session>
+  include <abstractions/bus-system>
+  include <abstractions/bus/org.freedesktop.login1>
+  include <abstractions/nameservice-strict>
+
+  capability fsetid,
+  capability fowner,
+  capability mknod,
+  capability chown,
+  capability setgid,
+  capability setpcap,
+  capability setuid,
+  capability fowner,
+  capability kill,
+
+  signal (send) peer=runsvdir,
+  signal (send) peer=runit,
+  signal (send) peer=NetworkManager,
+  signal (send) peer=agetty,
+  signal (send) set=(cont, term) peer=login,
+  signal (send) set=(cont, term) peer=dhcpcd,
+  signal (send) set=(cont, term) peer=sddm,
+  signal (send) peer=pause,
+  signal (send) peer=cupsd,
+  signal (send) peer=auditd,
+  signal (send) peer=chronyd,
+  signal (send) peer=dbus-daemon,
+  signal (send) peer=udevd,
+  signal (send) peer=rsyslogd,
+  signal (send) set=(kill, term) peer=startlxqt,
+  signal (send) set=(kill, term) peer=xorg,
+  signal (send) set=(term, cont) peer=@{p_systemd},
+  signal (receive) peer=runit,
+  signal (receive) peer=sddm,
+  signal (send) set=(cont, term) peer=elogind,
+  signal (receive) set=(cont, term) peer=elogind,
+  signal (receive) set=(hup) peer=@{p_systemd},
```

Comment:
As runsv is starts and manages the services, it is able to send signal everywhere: `signal (send),`